src/vscode/superbol-vscode-platform/superbol_vscode_platform.ml

```diff
@@ -230,14 +230,17 @@ let activate (extension : Vscode.ExtensionContext.t) =
       "superbol-free"
   in
 
-  Superbol_instance.set_bundled_superbol instance
-    bundled_superbol;
+  let instance = Superbol_instance.make ~bundled_superbol () in
+  current_instance := Some instance;
```

Comment:
May `activate` be called more than once with distinct `ExtensionContext`s? If yes, I guess that shall be handled later when we deal with storage URIs.

Reply:
No, `activate` is only called once (as is `deactivate`). Well, barring VS Code bugs, but that's the contract (cf https://code.visualstudio.com/api/references/activation-events#Start-up "An extension must export an activate() function from its main module and it will be invoked only once by VS Code when any of the specified activation events is emitted.").